packages/client/src/components/SearchFilter.vue

```diff
@@ -4,7 +4,7 @@
       <div class="align-self-end">
         <h4 class="mb-0">{{ selectedSearch === null ? "All Grants" : searchName }} </h4>
         <span v-if="selectedSearch !== null">
-          <a href="#" v-on:click.prevent="editFilter">Edit</a> | <a href="#" v-on:click.prevent="clearAll" v-if="$props.filterKeys.length > 0">Clear</a>
+          <a href="#" v-on:click.prevent="editFilter">Edit</a> | <a href="#" v-on:click.prevent="clearAll" >Clear</a>
```

Comment:
If you save a filter with only a name, the clear button wasn't displaying. Really this is bad data and validation should be added to prevent it, but until that's done, there's no reason the clear button shouldn't display.